### PR TITLE
fix: Fill in embedding and total parameters in ModelMeta

### LINKED ISSUE #3811: Update incorrect number of parameters for tomoro-colqwen3-embed-8b / -4b, ...
Some models have number of parameters which are not exact e.g. tomoro-colqwen3-embed-8b / -4b, ApsaraStackMaaS/EvoQwen2.5-VL-Retriever-7B-v1 / -3B-v1

We should report parameters the same way for consistency (originally mentioned in https://github.com/embeddings-benchmark/mteb/pull/3804#issuecomment-3697581076)

### PR DESCRIPTION
**Pull request by ayush1298**
This PR fills Total/Embedding parameters in ModelMeta for model listed under #3947 , so that we can calculate Active Parameters column on LB and merge PR https://github.com/embeddings-benchmark/mteb/pull/3979
Also, exact parameters are updated for each model, this will close #3811

### CODE REVIEW

**Comment by ayush1298** [comment]
Not done yet, filling all the Models. Will add results for other models and error/traces for models which I was not able to run.

File: mteb/models/model_implementations/jina_models.py [hunk 1]
```diff
@@ -826,7 +826,7 @@ def get_programming_task_override(
     revision="215a6e121fa0183376388ac6b1ae230326bfeaed",
     release_date="2024-09-18",  # official release date
     n_parameters=int(572 * 1e6),
-    n_embedding_parameters=None,
+    n_embedding_parameters=int(572 * 1e6),
```
[the remark this answers is not in the document]
Comment (ayush1298) [reply mteb/models/model_implementations/jina_models.py]: For Jina Models, I am not exactly sure, about these as I am not able to calculate them, So, should I do this or revert change here?

**Comment by ayush1298** [comment]
As I am updating parameters here, should I also resolve this issue:https://github.com/embeddings-benchmark/mteb/issues/3811,  and report exact parameters for all models, or should I do it in a separate PR?

**Comment by Samoed** [comment]
You can do this in this pr

**Comment by ayush1298** [comment]
I have also updated n_parameters to exact value. Below are the models, where I am seeing a very large change from old reported values. New parameters are taken from csv which I have mentioned here https://github.com/embeddings-benchmark/mteb/issues/3811#issuecomment-3756603406?

| model_name                                                  | status  | old_n_parameters | new_n_parameters | Did I check by manully Running
| ----------------------------------------------------------- | ------- | ---------------- | ---------------- | ---------------- |
| facebook/webssl-dino5b-full2b-224                           | updated | 5000000000       | 4938338816       | Yes |
| facebook/webssl-dino7b-full8b-224                           | updated | 7000000000       | 6450170880       | Yes |
| facebook/webssl-dino7b-full8b-378                           | updated | 7000000000       | 6452108288       | Yes |
| facebook/webssl-dino7b-full8b-518                           | updated | 7000000000       | 6454729728       | Yes |
| facebook/webssl-mae1b-full2b-224                            | updated | 1000000000       | 1136925696       | Yes |
| facebook/webssl-mae700m-full2b-224                          | updated | 700000000        | 632404480        | Yes |
| Alibaba-NLP/gte-Qwen2-1.5B-instruct                         | updated | 1780000000       | 1543268864       | Yes |
| Alibaba-NLP/gte-Qwen1.5-7B-instruct                  | updated | 7_720_000_000 | 7098994688       | Yes |
| Alibaba-NLP/gte-Qwen2-7B-instruct                           | updated | 7613000000       | 7069121024       | Yes
| Alibaba-NLP/gme-Qwen2-VL-7B-Instruct                        | updated | 8290000000       | 7746378240       | Yes |
| Salesforce/blip-image-captioning-base                       | updated | 247000000        | 223744258        | Yes (Seems this is still acceptable) |
| Salesforce/blip-image-captioning-large                      | updated | 470000000        | 446128642        | Yes (Seems this is still acceptable) |
| Salesforce/blip-itm-base-coco                               | updated | 247000000        | 223744258        | Yes (Seems this is still acceptable) |
| Salesforce/blip-itm-base-flickr                             | updated | 247000000        | 223744258        | Yes (Seems this is still acceptable) |
| Salesforce/blip-itm-large-coco                              | updated | 470000000        | 446128642        | Yes (Seems this is still acceptable) |
| Salesforce/blip-itm-large-flickr                            | updated | 470000000        | 446128642        | Yes (Seems this is still acceptable) |
| Salesforce/blip-vqa-base                                    | updated | 247000000        | 384672572        | Yes |
| Salesforce/blip-vqa-capfilt-large                           | updated | 247000000        | 384672572        |    Yes   |
| Qodo/Qodo-Embed-1-1.5B                                      | updated | 1780000000       | 1543268864       | Yes |
| GeoGPT-Research-Project/GeoEmbedding                        | updated | 7241732096       | 7110660096       | Yes |
| NovaSearch/jasper_en_vision_language_v1                     | updated | 1999000000       | 1564914432       | Yes |
| facebook/contriever-msmarco                                 | updated | 150000000        | 109482240        | Somehow Not able to run this now | 
| sentence-transformers/sentence-t5-xl                        | updated | 3000000000       | 1241696256       | Yes |
| llamaindex/vdr-2b-multi-v1                                  | updated | 2000000000       | 2208985600      | Yes |

File: mteb/models/model_implementations/jina_models.py [hunk 2]
```diff
@@ -776,8 +776,8 @@ def get_programming_task_override(
     revision="4a58ca57710c49f51896e4bc820e202fbf64904b",
     release_date="2025-06-24",  # official release date
     modalities=["image", "text"],
-    n_parameters=int(3.8 * 1e9),
-    n_embedding_parameters=None,
+    n_parameters=3934695552,
+    n_embedding_parameters=3934695552,
```
Comment (ayush1298) [inline mteb/models/model_implementations/jina_models.py]: I am not exactly sure about these, as I am not able to calculate them. So, should I do this or revert the change here?

Comment (KennethEnevoldsen) [reply mteb/models/model_implementations/jina_models.py]: This seems wrong - why are you not able to load this one?

Comment (KennethEnevoldsen) [reply mteb/models/model_implementations/jina_models.py]: For now you can set `n_embedding_parameters` to None and create a seperate issue on it, then we can take it in another PR

Comment (ayush1298) [reply mteb/models/model_implementations/jina_models.py]: I was able to load, but it doesn't support `get_input_embeddings()` which I was using to calculate embedding parameters. I have added code I used to calculate [here](https://github.com/embeddings-benchmark/mteb/issues/3947#issuecomment-3831121721)

Comment (ayush1298) [reply mteb/models/model_implementations/jina_models.py]: > For now you can set `n_embedding_parameters` to None and create a seperate issue on it, then we can take it in another PR

I have mentioned different errors that I was getting while running models [here](https://github.com/embeddings-benchmark/mteb/issues/3947#issuecomment-3830571044)

**Comment by ayush1298** [comment]
@Samoed, I have also updated [this table](https://github.com/embeddings-benchmark/mteb/pull/4031#issuecomment-3831187251). Should I go for new values, assuming that previously reported were incorrect, as I have checked each model by running again.

**Comment by Samoed** [comment]
It's better to fill with your new values

**Comment by ayush1298** [comment]
> It's better to fill with your new values

I have already added new values

**Comment by ayush1298** [comment]
I am not adding a test for checking` n_embedding_parameters is not None`, as we still have some models where we don't have these values, or if we have to add a case, then we have to create a list of excluded models in that case, which I think will not be a preferred way.

**Comment by KennethEnevoldsen** [comment]
> I am not adding a test for checking n_embedding_parameters is not None, as we still have some models where we don't have these values, or if we have to add a case, then we have to create a list of excluded models in that case, which I think will not be a preferred way.

I would add a test, but with exception (that we people can't add new models where it is None)

**Comment by ayush1298** [comment]
> > I am not adding a test for checking n_embedding_parameters is not None, as we still have some models where we don't have these values, or if we have to add a case, then we have to create a list of excluded models in that case, which I think will not be a preferred way.
> 
> I would add a test, but with exception (that we people can't add new models where it is None)

Then won't test will fail only as there are still models with `n_embedding_parameters=None`. Or, if are just keeping it in try-except then do I have to show list of models or something there?

**Comment by Samoed** [comment]
Should we add test in this PR then?

**Comment by ayush1298** [comment]
Should we add something like below for tests:

```
def test_n_embedding_parameters_coverage():
    """
    Test that tracks models with n_embedding_parameters=None.
    If any models are missing this field, the test is skipped with a warning message listing the affected models.
    """
    models_with_none = []
    for model_meta in mteb.get_model_metas():
        if model_meta.n_embedding_parameters is None:
            models_with_none.append(model_meta.name)

    if models_with_none:
        skip_message = (
            f"Models missing n_embedding_parameters ({len(models_with_none)}):\n"
        )
        pytest.skip(skip_message)
```

**Comment by Samoed** [comment]
Yes, you can add like this

File: tests/test_models/test_model_meta.py [hunk 3]
```diff
@@ -158,6 +158,23 @@ def test_fill_missing_parameter():
...
 
+def test_n_embedding_parameters_coverage():
+    """
+    Test that tracks models with n_embedding_parameters=None.
+    If any models are missing this field, the test is skipped with a warning message listing the no. of models with None value for n_embedding_parameters.
+    """
+    models_with_none = []
+    for model_meta in mteb.get_model_metas():
+        if model_meta.n_embedding_parameters is None:
+            models_with_none.append(model_meta.name)
```
Comment (Samoed) [inline tests/test_models/test_model_meta.py]: It's better to set list explicitly, otherwise newer models will pass the test

Comment (ayush1298) [reply tests/test_models/test_model_meta.py]: It's still a long list, all models which are still [here](https://github.com/embeddings-benchmark/mteb/issues/3947#issue-3821520888)

Comment (Samoed) [reply tests/test_models/test_model_meta.py]: This is fine. We have tests like https://github.com/embeddings-benchmark/mteb/blob/6fabfa0317cd28c4934e9d924c2423863bdb45c5/tests/test_tasks/test_metadata.py#L9 where we have long lists

Comment (ayush1298) [reply tests/test_models/test_model_meta.py]: oh, okay. Then will add that list in `test_model_meta` file only

Comment (ayush1298) [reply tests/test_models/test_model_meta.py]: Added.

File: tests/test_models/test_model_meta.py [hunk 4]
```diff
@@ -158,6 +343,25 @@ def test_fill_missing_parameter():
...
 
 
+@pytest.mark.parametrize("model_meta", mteb.get_model_metas())
+def test_n_embedding_parameters(model_meta: ModelMeta):
+    """
+    Test that tracks models with n_embedding_parameters=None.
+    Historic models (in _HISTORIC_MODELS) are allowed to have None values.
+    New models must have n_embedding_parameters defined, otherwise the test fails.
+    """
+    if model_meta.name in _HISTORIC_MODELS:
```
Comment (Samoed) [inline tests/test_models/test_model_meta.py:351]:
```suggestion
    if model_meta.name in _HISTORIC_MODELS:
        assert model_meta.n_embedding_parameters is None
```

Comment (Samoed) [reply tests/test_models/test_model_meta.py:351]: To make test fail if we fill models from historic

**Comment by ayush1298** [comment]
@Samoed Should I fix the broken links in the docs, documentation CI is failing because of it? 

```
WARNING -  Doc file 'index.md' contains a link './whats_new/2.0.md', but the target 'whats_new/2.0.md' is not found among documentation files.
WARNING -  Doc file 'index.md' contains a link './whats_new/2.0.md#upgrading-from-v1', but the target 'whats_new/2.0.md' is not found among documentation files.
WARNING -  Doc file 'whats_new.md' contains a link '../overview/available_models/text.md', but the target is not found among documentation files. Did you mean 'overview/available_models/text.md'?
```

File: tests/test_models/test_model_meta.py (hunk 4 above)
Comment (ayush1298) [reply tests/test_models/test_model_meta.py:351]: removed `pytest.skip` statement also from here

**Comment by Samoed** [comment]
I've fixed them https://github.com/embeddings-benchmark/mteb/pull/4043

File: tests/test_models/test_model_meta.py (hunk 4 above)
Comment (Samoed) [reply tests/test_models/test_model_meta.py:351]: Then test would fail. You need to add else or return here

Comment (ayush1298) [reply tests/test_models/test_model_meta.py:351]: Missed this one. Added it now

**Comment by ayush1298** [comment]
@KennethEnevoldsen Can we merge this one? Failed tests are already handled in other PR
